package-lock.json

```diff
@@ -1,19 +1,11055 @@
 {
   "name": "serverless-plugin-canary-deployments",
-  "version": "0.4.8",
-  "lockfileVersion": 1,
+  "version": "0.6.0",
```

Comment:
Ran `npm audit fix` since it was yeling at me.

![image](https://user-images.githubusercontent.com/18017094/131330840-72b43b4b-9c30-48c9-af40-a42975cf415a.png)